### ISSUE TITLE
French Dashboard page requires text to be translated

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#138]
URL:  [https://rsams.dev.cloud.openplus.ca/en/user/login](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Frsams.dev.cloud.openplus.ca%2Fen%2Fuser%2Flogin&data=04%7C01%7Cmanon.miron%40hc-sc.gc.ca%7Cec45cdb9ab0f4dfa788d08da02d39342%7C42fd9015de4d4223a368baeacab48927%7C0%7C0%7C637825404970600019%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=lo9rmCnhQp3Bhwvg5kSXBPsTI%2BEaKTdO2FReS2vb1Hs%3D&reserved=0)
User Agent: Chrome v99
Language: French

Issue Description: The text needs to match to that of the language of the page.  For any text that is not going to be translated a lang attribute (lang="en") will be required.

Steps to reproduce:
Test 1:
1. Log in to the French version of the app using the specified url above 
2. There are a number of English words that need to be translated on the Dashboard page: Dashboard, Title, Authored by, Authored on.  Additionally, the page title ‘User dashboard’ is in English.

Test 2:
1. Log in to the French version of the app using the specified url above 
2. From the ‘Tableau de bord’ page, select the ‘Recherche’ tab
4. The ‘Recherche dans les avis et les rappels’ has some issues with language of parts (e.g. Apply button, Status, Bookmark). I’m not sure if Admin can be translated.
5. Pick any of the titles under Titre and click on the ‘Reviser’ button
6. The ‘Message d'état’ page is displayed again and the following text needs to be translated in the 1st sentence under 'Message d'etat': 'New draft of' ‘has been created’.  Also, the period mid-sentence should be removed and is also an issue on the English page. 

Expected results: Text should be in the language of the page.
![image](https://user-images.githubusercontent.com/68117281/160182106-b07a132a-1bb8-462b-9d7b-01ee02f35f57.png)

![image](https://user-images.githubusercontent.com/68117281/160181341-e10c15de-8c28-4646-a22a-fe65dda2ed61.png)

![image](https://user-images.githubusercontent.com/68117281/160182246-3cf747fd-5567-422d-8f35-2b2dfc1fc073.png)

**Comment by duprasf** [comment]
Not sure why this was assigned to me, I'm not working on this project. Please explain what you would like me to do if it wasn't a mistake.

**Comment by mmiron-HC** [comment]
@duprasf this was done by accident.

**Comment by davidgotlieb** [comment]
For item #1, everything but the page title has been fixed on the prod CMS, you just aren't seeing it on Dev as this is a content issue.

For item 2 most have been fixed. I'll flag the rest. For the period after the title, that's a content issue as the program added it for some reason. Will flag to program but that is a user issue, not something I control.

**Comment by davidgotlieb** [comment]
Remaining French issues correct on Prod.
